src/command_modules/azure-cli-search/azure/cli/command_modules/search/tests/latest/test_adminkeys.py

```diff
@@ -0,0 +1,43 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.testsdk import ScenarioTest, ResourceGroupPreparer
+import unittest
+
+
+class AzureSearchAdminKeysTests(ScenarioTest):
+
+    @ResourceGroupPreparer(name_prefix='azure_search_cli_test')
+    def test_adminkeys_show_renew(self, resource_group):
+        self.kwargs.update({
+            'sku_name': 'standard',
+            'name': self.create_random_name(prefix='test', length=24),
+            'replica_count': 1,
+            'partition_count': 1,
+        })
+
+        self.cmd('az search services create -n {name} -g {rg} --sku {sku_name}',
+                 checks=[self.check('name', '{name}'),
+                         self.check('sku.name', '{sku_name}'),
+                         self.check('replicaCount', '{replica_count}'),
+                         self.check('partitionCount', '{partition_count}')])
+
+        _adminkey = self.cmd('az search adminkeys show -n {name} -g {rg}').get_output_in_json()
+        self.assertTrue(len(_adminkey['primaryKey']) > 0)
+        self.assertTrue(len(_adminkey['secondaryKey']) > 0)
+
+        self.cmd('az search adminkeys renew -n {name} -g {rg} --key-kind primary')
```

Comment:
`--key-kind` should be aliased to `--key`

Reply:
It's it confusing if:
in adminkeys, --key means key type (primary, secondary)
in querykeys, --key means key value
?

Maybe I should modify querykeys commands, any suggestion?

Reply:
Yes, for query-keys, use `--key` as an alias to `--key-kind` and `--key-value`. The reason is that many other services have these key type arguments and this is the pattern that has evolved. We want to keep them consistent across services.